src/icons/Reward.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+
+const Reward = ({ size = 34 }: { size?: number } = {}) => (
+  <Svg width={size} height={size} fill="none">
```

Comment:
does this need a viewbox too if you want to allow customizing size? Also I notice we use pngs for other similar icons. Is there a reason to use svg for this?

Reply:
Added `viewbox` in [e06a6a7](https://github.com/valora-inc/wallet/pull/6237/commits/e06a6a76dcc3ffc3d71f42b646b4d8c7fdd2b4ed), the reason I did an svg component instead of a png is that the image it was replacing was an svg component. I can change to a png instead if that is preferred though.